dev/com.ibm.ws.transport.http/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -95,6 +95,21 @@ http.writeTimeout.desc=Amount of time to wait on a socket for each portion of th
 http.removeServerHeader=Remove server header
 http.removeServerHeader.desc=Removes server implementation information from HTTP headers and also disables the default Liberty profile welcome page.
 
+http.limitNumHeaders=Limit number of headers
+http.limitNumHeaders.desc=Use to limit the number of HTTP headers that can be present in an incoming message.If this limit is exceeded, an error is returned to the remote client.
```

Comment:
Suggestion:
Limits the number of HTTP headers that can exist in an incoming message. <add a space> When this limit is exceeded, an error is returned to the remote client.